client/src/constants/validation.js

```diff
@@ -88,6 +88,7 @@ export const LoginSchema = yup.object().noUnknown().shape({
 });
 
 export const ApproveAccessRequestSchema = yup.object().noUnknown().shape({
+  sites: yup.array().required('Employer Sites is required').min(1, 'At least 1 employer site is required'),
```

Comment:
Should this only be required for Private Employer? Should only `region` be required for Health Authority role?

Reply:
Nope, required for both